sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -352,7 +352,6 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_sdw_machines[] = {
 		.link_mask = 0xF, /* 4 active links required */
 		.links = tgl_3_in_1_default,
 		.drv_name = "sof_sdw",
-		.sof_fw_filename = "sof-tgl.ri",
```

Comment:
what about other entries in the `snd_soc_acpi_intel_tgl_sdw_machines[]` array? Can they only be used on the LP variant? Would there be any harm if we removed FW filename from them too? Maybe from other architectures too? It's a nice simplification not to have to repeat the same code everywhere, let alone saving a 100 bytes of memory from remove a dozen of strings :-)

Reply:
@lyakh Good question. I talked with Bard when I made this patch and we decided to apply this patch only on the platforms which need patch, for safety. @plbossart @bardliao Do you have suggestion whether we should apply this patch on other platforms or only on this platform?

Reply:
@libinyang I agree with @lyakh. We can remove all `.sof_fw_filename` in soc-acpi-intel-tgl-match.c now, and review other platforms latter. In theory, we can remove them all except for special cases.